lab-geoff/lib/auth.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+let User = require('../model/model.js');
+
+module.exports = (req, res, next) => {
+  let auth = req.headers.authorization;
+  if(!auth) {
+    res.json({msg:'authentication headers needed'});
```

Comment:
You should import your error handling here and then pass a `createError()` into `next()` for the route to handle.